src/test/java/seedu/address/testutil/ZooKeepBookBuilder.java

```diff
@@ -6,7 +6,7 @@
 /**
  * A utility class to help with building ZooKeepBook objects.
  * Example usage: <br>
- *     {@code ZooKeepBook ab = new ZooKeepBookBuilder().withAnimal("John", "Doe").build();}
+ *     {@code ZooKeepBook ab = new ZooKeepBookBuilder().withAnimal(Harambe).build();}
```

Comment:
Parameter of withAnimal is an Animal object. Suggest to change "Harambe" to "harambe" to fit naming conventions.